scripts/cluster/redis-cli/on-start.sh

```diff
@@ -496,6 +497,27 @@ processRedisNode() {
     fi
 }
 
+loadInitData() {
+    if [ -d "/init" ]; then
+        log "INIT" "Init Directory Exists"
+        waitForAllRedisServersToBeReady 120
```

Comment:
Do we need to wait here ?  As is has been already called inside https://github.com/kubedb/redis-init-docker/blob/f7f3c43a3b89fbb33b4e40a572ac304b4f6dda0b/scripts/cluster/redis-cli/on-start.sh#L536

Reply:
Yes. Sometimes Redis servers become unresposive after adding nodes. So checking again before inserting data.